logicrunner/executionbroker.go

```diff
@@ -466,25 +469,18 @@ func (q *ExecutionBroker) finishPendingIfNeeded(ctx context.Context) {
 	}
 }
 
-func (q *ExecutionBroker) onPulseWeNotNext(ctx context.Context) (bool, []insolar.Message) {
+func (q *ExecutionBroker) onPulseWeNotNext(ctx context.Context) []insolar.Message {
 	logger := inslogger.FromContext(ctx)
 
 	q.stopRequestsFetcher(ctx)
 
 	messages := make([]insolar.Message, 0)
 	sendExecResults := false
-	active := q.isActive()
 
 	switch {
-	case active:
+	case q.IsActive():
```

Comment:
see comment in IsActive. just create isActive that has no lock and use it here and in public IsActive.

Reply:
made private